src/Evolution/Systems/RadiationTransport/M1Grey/Tags.hpp

```diff
@@ -109,6 +109,72 @@ struct TildeHSpatial : db::SimpleTag {
   }
 };
 
+/// The emissivity of the fluid for neutrinos of a given species
+/// By convention, this is the emitted energy per unit volume.
+template <class Species>
+struct GreyEmissivity : db::SimpleTag {
+  using type = Scalar<DataVector>;
+  static std::string name() noexcept {
+    return "GreyEmissivity_" + neutrinos::get_name(Species{});
+  }
+};
+
+/// The absorption opacity of the fluid for neutrinos of a
+/// given species.
+/// As c=1, this is both the absorption per unit length
```

Comment:
add `///` line to separate the brief comment from the details (for all of the tags)